qna/DE-certified-online-TSE.md

```diff
@@ -1,8 +1,24 @@
 ## Question
-Is there any certified online TSE available for germany?
+Is there any certified Cloud TSE available for Germany?
+
+## Question
+
+I want to roll out with a Cloud TSE and  I do not want to wait any longer. What options do I have?
+
+## Question
+
+I have ordered a Cloud TSE from my dealer. Can I use an uncertified Cloud TSE in my region until a certified version is available?
 
 ## Metadata tags
 lang-en, market-de, middleware, PosCreator, PosDealer, Consultant
 
 ## Answer
-Currently (18.08.2020), there is **no certified** online TSE available for the german market. 
+Currently (23.09.2020), there is **no certified** Cloud TSE available for the german market. fiskaltrust will offer certified Cloud TSE's in the production environment as soon as they are available on the market. In the meanwhile, we provide uncertified Cloud TSEs for integration and testing in the sandbox environment.
+
+PosDealers can already order fiskaly Cloud TSEs from fiskaltrust by [signing a frame contract.](https://github.com/fiskaltrust/productdescription-de-doc/blob/8a9c3fa62c9dd0c38662c7127a724162703242e3/for-posdealers/02-pre-sales/01-purchase-agreement.md)
+
+If a PosOperator wants to be compliant from now on in a production environment, we recommend to use certified Hardware TSE's.
+
+In general, if PosOperators are using an uncertified TSE in production environment or no TSE at all, it will both result in generating POS-receipts which are not compliant to German law. In most regions of Germany it may be possible to [extend the "Nichtbeanstandungsregelung" under some specific circumstances](https://github.com/fiskaltrust/faq/blob/master/qna/DE-Nichtbeanstandungsregelung.md#regionale-erg%C3%A4nzungen-zur-nichtbeanstandungsregelung). We recommend to refer to a tax advisor to ensure to fulfil all requirements to be fully compliant with the German law.
+
```

Comment:
Please add: "Please be also aware of the regulation that it is not allowed to offer or place a uncertified TSE on the market".